### PR TITLE
add Simplified Chinese(zh_CN) translation

### PR DESCRIPTION
**Pull request by VebinLee**
add Simplified Chinese(zh_CN) translation

### CODE REVIEW

**Comment by fabaff** [comment]
Related #33

**Review by domingoruiz — APPROVED** [review]

**Comment by VebinLee** [comment]
add zh_CN to head.html files @domingoruiz

File: vercel.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   },
   "routes": [
     { "handle": "filesystem" },
+    { "src": "/zh_CN/(.*)", "status": 404, "dest": "/zh_CN/404.html" },
```
Comment (domingoruiz) [inline vercel.json]: We only use the first two digits of the language for the browser's self-recognition to work. Change zh_CN to zh.

File: i18n/zh_CN.toml [hunk 2]
```diff
@@ -0,0 +1,90 @@
+[language]
+other = "zh_CN"
```
Comment (domingoruiz) [inline i18n/zh_CN.toml]: We only use the first two digits of the language for the browser's self-recognition to work. Change zh_CN to zh.

File: config.toml.bak [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    weight = 4
+  [languages.ru]
+    languageName = "Русский"
+    title = "Rocky Linux"
+    weight = 5
+  [languages.lv]
+    languageName = "Latviski"
+    title = "Rocky Linux"
+    weight = 6
+  [languages.zh_CN]
```
Comment (domingoruiz) [inline config.toml.bak]: We only use the first two digits of the language for the browser's self-recognition to work. Change zh_CN to zh.

File: i18n/zh_CN.toml (hunk 2 above)
Comment (domingoruiz) [inline i18n/zh_CN.toml]: Also rename this file to zh.toml

File: config.toml [hunk 4]
```diff
@@ -36,28 +36,11 @@ DefaultContentLanguage = "en"
...
-    weight = 4
-  [languages.ru]
-    languageName = "Русский"
-    title = "Rocky Linux"
-    weight = 5
-  [languages.lv]
-    languageName = "Latviski"
-    title = "Rocky Linux"
-    weight = 6
+  [languages.zh_CN]
```
Comment (domingoruiz) [inline config.toml]: We only use the first two digits of the language for the browser's self-recognition to work. Change zh_CN to zh.

**Review by domingoruiz — CHANGES_REQUESTED** [review]
We only use the first two digits of the language for the browser's self-recognition to work. Change zh_CN to zh.

File: vercel.json (hunk 1 above)
Comment (VebinLee) [reply vercel.json]: Change zh_CN to zh

File: i18n/zh_CN.toml (hunk 2 above)
Comment (VebinLee) [reply i18n/zh_CN.toml]: Change zh_CN to zh

File: config.toml.bak (hunk 3 above)
Comment (VebinLee) [reply config.toml.bak]: Change zh_CN to zh

Comment (oubayun) [reply config.toml.bak]: Change zh_CN to zh

File: i18n/zh_CN.toml (hunk 2 above)
Comment (VebinLee) [reply i18n/zh_CN.toml]: Change zh_CN to zh

File: config.toml (hunk 4 above)
Comment (VebinLee) [reply config.toml]: Change zh_CN to zh

**Review by domingoruiz — COMMENTED** [review]
Ok good

**Pull request merged**